README.md

```diff
@@ -591,6 +591,26 @@ client.index('myIndex').updateSettings(settings: Settings): Promise<EnqueuedTask
 client.index('myIndex').resetSettings(): Promise<EnqueuedTask>
 ```
 
+### Pagination Setting
+
+#### [Get pagination settings](https://docs.meilisearch.com/reference/api/settings.html#get-pagination-settings)
+
+```ts
+client.index('myIndex').getPagination(): Promise<PaginationSettings>
```

Comment:
`PaginationSettings` should exists but doesnt for the moment

Reply:
> `PaginationSettings` should exists but doesnt for the moment

https://github.com/meilisearch/meilisearch-js/blob/main/src/types/types.ts#L188